tooling/internal/cmd/generatecommand.go

```diff
@@ -37,9 +37,14 @@ func newGenerateCommand() *cobra.Command {
 		Long:                  `generate code for the package in the current directory`,
 		DisableFlagsInUseLine: true,
 		Args:                  cobra.NoArgs,
-		Run: func(*cobra.Command, []string) {
+		Run: func(cmd *cobra.Command, args []string) {
+			configOverrides, err := cmd.Flags().GetStringArray("config")
```

Comment:
Did you consider using `cmd.Flags().StringToStringVar`? The param will be in the help text and will be parsed already as a `map[string]string`

Reply:
It was already in the help text correctly (`  -c, --config key=value   Override key=value pair in "_package.yml"`), but your implementation in Tyger is much simpler. I will clean this up.